doc/run_on_kubernetes.md

```diff
@@ -0,0 +1,59 @@
+# Running SQLFlow on Kubernetes
+
+This is a tutorial on how to run SQLFlow on Kubernetes, and this tutorial will deploy:
+- MySQL server with the SQLFlow example datasets loaded, and
+- SQLFlow components contain the sqlflow-server the Jupyter notebook.
+
+Then you can run the SQLFlow query in the Jupyter notebook on your web browser.
+
+## Preparation
+
+1. Setup a Kubernetes cluster: You can refer to the [official page](https://kubernetes.io/docs/setup) to set up a 
+full cluster or use a local quick start tool: [minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/)
+This tutorial would use [minikube] to demonstrate the SQLFlow.
+1. [Install the kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/), which is the command line tool
+to interact with the Kubernetes cluster.
+1. Pull the official Docker image [sqlflow/sqlflow:latest](https://hub.docker.com/r/sqlflow/sqlflow) or [build
```

Comment:
I am not sure if we need to docker pull the Docker image to the local computer. I think we need to make sure that the Kubernetes nodes, but not our local computer, could pull the image.

Reply:
Thanks for your remind, you're right, we don't need to pull the Docker image on the host.